src/views/product-landing/server.ts

```diff
@@ -26,12 +36,48 @@ async function generateStaticProps({
   // TODO: content is accurate, while not needing to be
   // TODO: manually kept up to date?
   // Asana task: https://app.asana.com/0/1201010428539925/1201646299837754/f
+  const usedHeadings = []
   CONTENT.blocks = CONTENT.blocks.map((block) => {
-    const { type, heading } = block
-    if (type !== 'heading') return block
-    return {
-      ...block,
-      slug: slugify(heading, { lower: true }),
+    switch (block.type) {
+      case 'heading':
+        // augment heading blocks with a consistent slug,
+        // used for anchor linking
+        // if the resulting slug is not unique, append an
+        // integer suffix to make sure that it is
+        // eslint-disable-next-line no-case-declarations
+        const baseSlug = slugify(block.heading, { lower: true })
+        // eslint-disable-next-line no-case-declarations
+        let slug = baseSlug
+        // eslint-disable-next-line no-case-declarations
+        let slugMakeUniquePrefix = 0
+        while (usedHeadings.indexOf(slug) !== -1) {
+          slugMakeUniquePrefix++
+          slug = `${baseSlug}-${slugMakeUniquePrefix}`
+        }
+        usedHeadings.push(slug)
+        return { ...block, slug }
+      case 'cards':
+        // TODO: instead of the below, should likely use product context
```

Comment:
Added a note here on using product context. Intent is to leave this for a follow up PR, focus right now is on getting the placeholder content in front of product stakeholders for review.